hummingbot/connector/exchange/kraken/kraken_order_book_tracker.py

```diff
@@ -45,7 +48,7 @@ def __init__(self,
     @property
     def data_source(self) -> OrderBookTrackerDataSource:
         if not self._data_source:
-            self._data_source = KrakenAPIOrderBookDataSource(trading_pairs=self._trading_pairs)
+            self._data_source = KrakenAPIOrderBookDataSource(trading_pairs=self._trading_pairs, api_factory=self._api_factory)
```

Comment:
this line is over 120 characters
```suggestion
            self._data_source = KrakenAPIOrderBookDataSource(
                trading_pairs=self._trading_pairs, api_factory=self._api_factory
            )
```